tests/admin.test.ts

```diff
@@ -71,28 +71,33 @@ describe('retroactive attendance submission', () => {
 
     await adminController.submitAttendanceForUsers(
       { users: [user.email], event: event.uuid },
-      proxyUser,
+      admin,
     );
 
-    const userResponse = await userController.getUser({ uuid: user.uuid }, proxyUser);
-    const attendanceResponse = await attendanceController.getAttendancesForCurrentUser(user);
-    const activityResponse = await userController.getCurrentUserActivityStream(user);
-
+    const userResponse = await userController.getUser({ uuid: user.uuid }, admin);
     expect(userResponse.user.points).toEqual(user.points);
+
+    const attendanceResponse = await attendanceController.getAttendancesForCurrentUser(user);
     expect(attendanceResponse.attendances).toHaveLength(1);
+    expect(attendanceResponse.attendances[0].event.uuid).toEqual(event.uuid);
+    expect(attendanceResponse.attendances[0].asStaff).toEqual(false);
```

Comment:
What's this assertion for? This doesn't seem useful and can be removed.

Reply:
As per the previous review, this verifies the attendance response matches what we'd expect to receive when attending a specific event.

Reply:
Ah right. LGTM but one of the tests is failing.